README.md

```diff
@@ -2,15 +2,32 @@
 
 Scalable Object Ledger and CDN.
 
-## Quick Start
+## Introducing Warehouse 7
+
+Warehouse 7, also known as Warehouse Next, is a scalabe object ledger and cdn system for powering dynamic web applications. The new system has been entirly redisgned using a lightweight, portable and flexible architecture.
+
+While designing the new system, it made sense to add a higher level of abstraction.
+The idea is expose a generic Object API and a CDN service. The user can use the CDN service to upload the desired assets while using the Object API to store the metadata of the assets (e.g., file urls, etc.)
+
+The Object API acts as generic ledger and it can be used by a third-party system to store any type of data.
+
+Warehouse can be used for different use cases in different ways. At GoDaddy we mainly use it for storing and serving our web applications compiled assets. Our apps workflow can be summarized as following:
+
+1. Engineers create a new service or package version and release it (e.g., `npm publish`, `git push <tag>`)
+1. A CICD workflow is triggered and runs the Webpack builds
+1. When all builds are completed, the CICD workflow uploads the assets to Warehouse
+
+## Get Started
+
+### Quick Start
 
 ```bash
 git clone git@github.com:godaddy/warehouse.ai.git
 cd warehouse.ai && npm install
 npm start
```

Comment:
Is there needed DynamoDB env config before starting?

Reply:
Nope, needs localstack only